### PR TITLE
Update l10nzhtw.properties #603

### PR DESCRIPTION
**Pull request by JE-Chen**
#603

### CODE REVIEW

File: l10nzhtw.properties [hunk 1]
```diff
@@ -1,14 +1,20 @@
...
 
 defaults.mavenGroupId=org.codehaus.sonar-plugins.l10n
-defaults.mavenArtifactId=sonar-l10n-zhtw-plugin
+defaults.mavenArtifactId=sonar-l10n-zh-tw-plugin
+
+1.1.description=SonarQube Traditional Chinese Pack Support SonarQube 10.* Version
+1.1.sqVersions=[10.0,LATEST]
+1.1.date=2024-10-21
+1.1.changelogUrl=https://github.com/JE-Chen/sonar-l10n-zh-tw/releases/tag/1.1
+1.1.downloadUrl=https://github.com/JE-Chen/sonar-l10n-zh-tw/releases/download/1.1/sonar-l10n-zh-tw-plugin.jar
```
Comment (ganncamp) [inline l10nzhtw.properties:14]: This is 404

**Review by ganncamp — CHANGES_REQUESTED** [review]

File: l10nzhtw.properties (hunk 1 above)
Comment (JE-Chen) [reply l10nzhtw.properties:14]: Ah, sorry. I changed the jar file, so the old URL returns a 404 error.

Comment (JE-Chen) [reply l10nzhtw.properties]: Done

File: l10nzhtw.properties [hunk 2]
```diff
@@ -1,14 +1,20 @@
...
 
 defaults.mavenGroupId=org.codehaus.sonar-plugins.l10n
-defaults.mavenArtifactId=sonar-l10n-zhtw-plugin
+defaults.mavenArtifactId=sonar-l10n-zh-tw-plugin
+
+1.1.description=SonarQube Traditional Chinese Pack Support SonarQube 10.* Version
+1.1.sqVersions=[10.0,LATEST]
+1.1.date=2024-10-21
+1.1.changelogUrl=https://github.com/JE-Chen/sonar-l10n-zh-tw/releases/tag/1.1
+1.1.downloadUrl=https://github.com/JE-Chen/sonar-l10n-zh-tw/releases/download/1.1/sonar-l10n-zh-tw-plugin-1.1.jar
```
Comment (ganncamp) [inline l10nzhtw.properties:14]: Okay, the download works now, but my server won't start 
```
Caused by: java.lang.IllegalStateException: The plugin [l10nzhtw] does not support Java 17.0.12
...
Caused by: java.lang.UnsupportedClassVersionError: org/sonar/plugins/l10n/TraditionalChinesePackPlugin has been compiled by a more recent version of the Java Runtime (class file version 67.0), this version of the Java Runtime only recognizes class file versions up to 61.0
```
Current SonarQube versions support / use Java 17

**Review by ganncamp — CHANGES_REQUESTED** [review]

File: l10nzhtw.properties (hunk 2 above)
Comment (JE-Chen) [reply l10nzhtw.properties:14]: {75DD9D23-2F8C-4B57-AC18-2E19F979D4BA}
I used the wrong JDK, I will rebuild the jar.

Comment (JE-Chen) [reply l10nzhtw.properties:14]: I have already updated the jar file built using JDK 17.

Comment (JE-Chen) [reply l10nzhtw.properties:14]: Same name & same Url

**Comment by ganncamp** [comment]
I couldn't make my browser switch to Chinese, but I assume you've tested locally, so you're in. Congrats

**Pull request merged**